admin/database/database.go

```diff
@@ -540,6 +546,19 @@ type DeviceAuthCode struct {
 	UpdatedOn     time.Time           `db:"updated_on"`
 }
 
+type AuthorizationCode struct {
```

Comment:
nit: Add a small comment clarifying it's usage (i.e. that it is used for the OAuth PKCE flow)